colossalai/checkpoint_io/utils.py

```diff
@@ -6,9 +6,9 @@
 from colossalai.tensor.d_tensor.d_tensor import DTensor
 
 SAFE_WEIGHTS_NAME = "model.safetensors"
-WEIGHTS_NAME = "model.bin"
+WEIGHTS_NAME = "pytorch_model.bin"
 SAFE_WEIGHTS_INDEX_NAME = "model.safetensors.index.json"
-WEIGHTS_INDEX_NAME = "model.bin.index.json"
+WEIGHTS_INDEX_NAME = "pytorch_model.bin.index.json"
```

Comment:
Assume somehow the checkpoint name does not start with `pytorch_model`, does it still work?

Reply:
it can't load hugging face model,   works well in loading the model saved by colossalai.